### PR TITLE
Update change version to use apiv4

### PR DESCRIPTION
**Pull request by hatched**
Old change-version generated the version numbers but charmstore apiv4 now has them available.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2427/
Test PASSed.

File: app/store/charmstore-api.js [hunk 1]
```diff
@@ -407,6 +407,53 @@ YUI.add('charmstore-api', function(Y) {
...
+      @method getAvailableVersions
+      @param {String} charmId The charm id to fetch all of the versions for.
+      @param {Function} successCallback The success callback.
+      @param {Function} failureCallback The failure callback.
+    */
+    getAvailableVersions: function(charmId, successCallback, failureCallback) {
+      charmId = charmId.replace('cs:', '');
+      var series = charmId.split('/')[0];
+      this._makeRequest(
+          this._generatePath(charmId, null, '/expand-id'),
```
Comment (huwshimi) [inline app/store/charmstore-api.js]: Unless I'm misunderstanding how this function is to be used it looks like you're not actually using the result of this function call, could that be right?

**Comment by huwshimi** [comment]
When doing a QA in the sandbox environment I have the following issue:
- add a service, deploy
- click "Change version"
- click "Upgrade" for any version
- in the inspector header the version won't change
- closing and opening the inspector or clicking on the service block will update the version number correctly
- now click "Upgrade" for a different version and the header will update correctly

File: app/store/charmstore-api.js (hunk 1 above)
Comment (hatched) [inline app/store/charmstore-api.js]: The return value of this function is being passed to the first argument of the `_makeRequest` function

**Comment by hatched** [comment]
Thanks for the review and qa @huwshimi That bug you found also appears in trunk so I'll make a bug for a follow-up

**Comment by makyo** [comment]
:+1: much simpler than before, thank you!  No QA yet.

**Comment by hatched** [comment]
Thanks for the reviews! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**